README.md

```diff
@@ -28,4 +28,48 @@ armor_monoid.register_armor_group("arcane", 100)
 <br/>
 As you can see, the argument is not a multiplier, but the base armor group
 rating. Calling this would mean players start off with an armor rating in
-"arcane" of 100 (no protection).
\ No newline at end of file
+"arcane" of 100 (no protection).
+
+Special armor groups
+====================
+
+Luanti defines a number of [special armor groups](https://github.com/luanti-org/luanti/blob/master/doc/lua_api.md#objectref-armor-groups)
+that have an engine-based effect and therefore must be handled uniquely by this
+monoid.
+
+`fall_damage_add_percent`
+-------------------------
+
+The armor monoid handles this group exactly like normal armor groups that have a
+base value of 100, but because its value is multiplicative with built-in fall
+damage, its final value is internally offset by -100 to respect this behavior.
+
+To grant a player fall damage reduction, use the `fall_damage_add_percent` group
+as you would any normal damage group:
+
+```
```

Comment:
Please specify `lua` for syntax highlighting on all relevant code blocks to make it more pleasant to the reader:
```
\`\`\`lua
```
(ignore the backticks. I cannot provide a code suggestion properly because escaping it does apparently not work correctly.)

Reply:
Done!